mininet/link.py

```diff
@@ -194,6 +240,7 @@ def config( self, mac=None, ip=None, ifconfig=None,
         r = {}
         self.setParam( r, 'setMAC', mac=mac )
         self.setParam( r, 'setIP', ip=ip )
+        self.setParam( r, 'setIP', ip=ip6 )
         self.setParam( r, 'isUp', up=up )
```

Comment:
Add an option to set an IPv6 address with keyword argument.  Backwards compatible.